csrc/scheduler/utils.cpp

```diff
@@ -997,9 +997,8 @@ std::vector<TensorView*> cacheInputs(Fusion* fusion, bool unroll) {
   // If we're going to unroll, make a cache of the inputs
   auto in_tvs = ir_utils::filterByType<TensorView>(fusion->inputs());
   for (auto tv : in_tvs) {
-    if (tv->uses().empty() || ir_utils::isTorchGatherIndicesTv(tv) ||
-        ir_utils::isTorchGatherLookupTv(tv) || ir_utils::isSelectInput(tv) ||
-        ir_utils::isIndexSelectLookupTv(tv)) {
+    if (tv->uses().empty() || ir_utils::isTorchGatherLookupTv(tv) ||
+        ir_utils::isSelectInput(tv) || ir_utils::isIndexSelectLookupTv(tv)) {
```

Comment:
Removed `isTorchGatherIndicesTv` as it shouldn't be necessary

Reply:
`isTorchGatherIndicesTv` is no longer used anymore, should we remove it? I can not think of a case where the the index tv is special.

Reply:
Removed